sktime/transformations/series/boxcox.py

```diff
@@ -266,22 +279,18 @@ def _transform(self, X, y=None):
         """
         from scipy.special import boxcox
 
-        enforce_positive = self.enforce_positive
-        lambda_ = self.lambda_
-
         X_shape = X.shape
         X = X.flatten()
 
-        if enforce_positive:
+        if self.enforce_positive:
             X_sign = np.sign(X)
 
-        Xt = boxcox(np.abs(X), lambda_)
+        Xt = boxcox(np.abs(X), self.lambda_)
 
-        if enforce_positive:
+        if self.enforce_positive:
             Xt = Xt * X_sign
 
-        Xt = Xt.reshape(X_shape)
```

Comment:
similarly, this is an "explicit code style" choice, to have a line in wihch the returned object is assigned. Besides for readability, this is helpful for debugging as well, since a checkpoint can be set here.

Could you please revert any changes from the explicit code style to the more obscure code style? Thanks!